drivers/soundwire/Makefile

```diff
@@ -16,7 +16,7 @@ soundwire-bus-y += debugfs.o
 endif
 
 #AMD driver
-soundwire-amd-y :=	amd_manager.o
+soundwire-amd-y := amd_init.o amd_manager.o
```

Comment:
the commit message is not helping much, it's not self-explanatory for someone without access to internal documentation. Please try to uplevel for non-AMD reviewers :-)

Reply:
IMHO, In the commit message, at high level below content will be enough.

Implementation of generic functions to have a common functionality for
SoundWire manager platform device creation, start and exit sequence
for both legacy(NO DSP) and SOF stack for AMD platforms.
These functions will be invoked from legacy and SOF stack.

Reply:
IMHO, below commit message will be enough at higher level.

Implementation of generic functions to have a common functionality for
SoundWire manager platform device creation, start and exit sequence
for both legacy(NO DSP) and SOF stack for AMD platforms.
These functions will be invoked from legacy and SOF stack.